htdocs/core/boxes/box_funnel_of_prospection.php

```diff
@@ -97,6 +97,8 @@ public function loadBox($max = 5)
 		$sql = "SELECT cls.rowid, cls.code, cls.percent, cls.label";
 		$sql .= " FROM " . MAIN_DB_PREFIX . "c_lead_status as cls";
 		$sql .= " WHERE active=1";
+		$sql .= " Order by cls.rowid";
+		$sql .= " AND cls.code != 'LOST'";
```

Comment:
the different operator for SQL is <>   ( !=  is supported but not standard)

Reply:
it's fixed